sceptre/scipool/README.md

```diff
@@ -31,15 +31,63 @@ Requirements:
 * Run `pre-commit install` to install the git hook.
 
 ## Testing
+
+### Pre-Commit
 As a pre-deployment step we syntatically validate our sceptre and
 cloudformation yaml files with [pre-commit](https://pre-commit.com).
 
 Please install pre-commit, once installed the file validations will
-automatically run on every commit.  Alternatively you can manually
+automatically run on every commit. Alternatively you can manually
 execute the validations by running `pre-commit run --all-files`.
 Please install pre-commit, once installed the file validations will
 automatically run on every commit.
 
+### Pre-Merge Integration Testing
+
+To validate the integration of AMI changes with Service Catalog products several
+resources need to be created in AWS including a test AMI, a test CFN template
+(uploaded to S3), and a test product in Service Catalog. Finally, the test
+product can be provisioned from https://sc-dev.sageit.org/ for user-acceptance
+testing of the changes to the AMI and/or CFN template.
+
+Anyone with write access to the upstream repos can automate creating AMIs and
+uploading CFN templates to S3 by pushing `test/*` branches directly to the
+uptsream repo. For example, pushing a branch named `test/foo` directly to
+`Sage-Bionetworks-IT/packer-rstudio` will create an AMI named
+`packer-rstudio-test/foo` in `imagecentral`, and pushing a branch named
+`test/bar` directly to `Sage-Bionetworks/service-catalog-library` will upload
+the templates to `service-catalog-library/test/bar/` in the bootstrap bucket in
+`admincentral`.
+
+The standard PR development process is used to create a test product in Service
+Catalog in this `scipool` sceptre project, such as in [this PR](https://github.com/Sage-Bionetworks-IT/organizations-infra/pull/1109).
+
+The full process with automation is:
+
+1. Commit changes to the target packer repo to modify the AMI on a branch that
+   starts with `test/`, and push it directly to the upstream repo to create a
+   test AMI.
+   - Initial manual testing of the AMI can be performed by using it to boot an
+     EC2 instance.
+1. Commit changes to `service-catalog-library` on a branch that starts with
+   `test/` to update the relevant template with the new AMI, and push directly
+   to the origin repo to uplead the templates to S3.
+1. Create a pull request for `organizations-infra` to add a new Service Catalog
```

Comment:
I think there should be some mention that to add or change a product in sc-dev the change goes in sceptre/scipool/config/develop. Later, to "roll out" the product change to production, the change is mirrored to sceptre/scipool/config/prod, ../strides, and ../bmgf-ki.

Reply:
great suggestion, thanks